README.md

```diff
@@ -294,6 +296,13 @@ themeit.createThemes({
 themeit.applyTheme("nested-light-theme");
 ```
 
+## Examples
+As a way of show casing the power of themeit we created some examples in different frameworks.
+The examples are just as a showcase. Themeit doesn't, in any way, depend or the frameworks below.
+
+### Vue
+A basic Vue example of Theme It can be found here: https://codesandbox.io/s/zz2on366jl?view=preview
```

Comment:
If you could just update this to be a link to your example that would be great. 

Like this - [Vue](https://codesandbox.io/s/zz2on366jl?view=preview)